src/scancode/agent/scancode_wrapper.cc

```diff
@@ -70,6 +70,7 @@ void scanFileWithScancode(const State &state, string fileLocation, string output
 
   string command =
     "PYTHONPATH='/home/" + projectUser + "/pythondeps/' " +
+    "SCANCODE_CACHE='/usr/local/share/fossology/scancode/agent/.cache' " +
```

Comment:
Ummm... Please do not hardcode the path like this here. It will change to `/usr/share/fossology` if installed as a `.deb` package.

Instead, I will recommend to use the `/home/<projectUser>/.cache` as it reads the username from config and is dynamic enough.

Alternatively, this variable can be moved to the `runscanonfiles.py` file and exported using `os.environ["SCANCODE_CACHE"]` and `__file__` variables.

Reply:
Yea I shortly realized hardcoding path isn't the best way (or worst i'll say 🙂). Will make the changes soon.